cypress/integration/console/integrations/webhooks/create-with-template.spec.js

```diff
@@ -0,0 +1,104 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+describe('Application Webhook create', () => {
+  const userId = 'create-app-test-user'
+  const user = {
+    ids: { user_id: userId },
+    primary_email_address: 'create-app-test-user@example.com',
+    password: 'ABCDefg123!',
+    password_confirm: 'ABCDefg123!',
+  }
+  const appId = 'webhook-test-application'
+  const application = {
+    ids: {
+      application_id: appId,
+    },
+  }
+
+  before(() => {
+    cy.dropAndSeedDatabase()
+    cy.createUser(user)
+    cy.createApplication(application, userId)
+  })
+
+  beforeEach(() => {
+    cy.loginConsole({ user_id: userId, password: user.password })
+    cy.fixture('console/application/integrations/webhook/template.json').then(webhookTemplate => {
+      cy.intercept(
+        'GET',
+        `/api/v3/as/webhook-templates?field_mask=base_url,create_downlink_api_key,description,documentation_url,downlink_ack,downlink_failed,downlink_nack,downlink_queue_invalidated,downlink_queued,downlink_sent,fields,format,headers,ids,info_url,join_accept,location_solved,logo_url,name,service_data,uplink_message`,
+        webhookTemplate,
+      )
```

Comment:
```suggestion
      cy.intercept(
        'GET',
        `/api/v3/as/webhook-templates?field_mask=base_url,create_downlink_api_key,description,documentation_url,downlink_ack,downlink_failed,downlink_nack,downlink_queue_invalidated,downlink_queued,downlink_sent,fields,format,headers,ids,info_url,join_accept,location_solved,logo_url,name,service_data,uplink_message`,
        { fixture: 'console/application/integrations/webhook/template.json' },
      )
```

Intercept can also ingest the fixture directly.